src/backend/binaryen/wasm_expr_gen.ts

```diff
@@ -2033,150 +2120,142 @@ export class WASMExpressionGen {
     private getInstMember(
         thisRef: binaryen.ExpressionRef,
         ownerType: ObjectType,
-        meta: ObjectDescription,
-        member: MemberDescription,
+        typeMeta: ObjectDescription,
+        typeMember: MemberDescription,
         isCall = false,
         args?: SemanticsValue[],
+        isSetter = false,
     ) {
         const thisTypeRef = this.wasmTypeGen.getWASMType(ownerType);
-        const valueIdx = this.getTruthIdx(meta, member);
-        const typeMember = meta.findMember(member.name) as MemberDescription;
-        if (meta.isInterface) {
+        const valueIdxInTypeMeta = this.getTruthIdx(
+            typeMeta,
+            typeMember,
+            isSetter,
+        );
+        const metaRef = FunctionalFuncs.getWASMObjectMeta(this.module, thisRef);
+        const memberNameRef = this.getStringOffset(typeMember.name);
+        const indexRef = this.getPropIndexFromObj(
+            metaRef,
+            memberNameRef,
+            ItableFlag.UNKNOWN,
+        );
+        const flagRef = this.getPropFlagFromObj(
+            metaRef,
+            memberNameRef,
+            ItableFlag.UNKNOWN,
+        );
```

Comment:
These are only used for interface, right? Then had better move them into the if branch

Reply:
Yes, get it.